src/wmnds/components/content-tiles/travel-tips/travel-tips.njk

```diff
@@ -6,53 +6,37 @@
     <p>You will still be able to get to where you need to go, but consider the following:</p>
   </div>
 
-  <div class="wmnds-msg-help wmnds-content-tile--purple">
-  {% from "wmnds/components/message/help/_help.njk" import wmndsMsgHelp %}
-  {{
-    wmndsMsgHelp({
-        message: '<span>Plan ahead and leave more time for your journey:</span> If you travel in the Five Ways area you’ll need more time for your journey to take account of the closure of Five Ways underpass and changes along Broad Street.'
-      })
-  }}
+  <div class="wmnds-content-tile--purple">
```

Comment:
```suggestion
  <div class="wmnds-content-tile--block">
```

Change all instances in this file to the new class so it matches the new class we defined in the scss file.